packages/ui/src/theme/css/component/authenticator.scss

```diff
@@ -139,3 +139,32 @@
     padding: 0px 1.5rem;
   }
 }
+[data-amplify-copy] {
+  display: flex;
+  font-size: 0.688rem;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+
+  @media (max-width: 30rem) {
+    word-break: break-all;
+  }
```

Comment:
What do you think about making `[data-amplify-copy]` `hidden` if not on mobile?

Reply:
I dunno, I was just using a QR code the other day, and I appreciated how the secret key was also displayed. I don't think it has to be just for mobile.

Reply:
That's fair! My only concern is that end-users won't know what to do with that secret key, but customers can use header slots to add directions.

Reply:
Yeah, I was debating back and forth if I should add in some instructions on how to add 2FA to Authy, or Google Authenticator, but I didn't want to add more requirements unless a customer wanted too.

And they can add in their own with a slot, so you're right.